dotCMS/src/main/java/com/dotcms/rendering/velocity/directive/ContentletDetail.java

```diff
@@ -32,20 +33,25 @@ public String getName() {
 
 
     private long resolveLang(final String identifier, final RenderParams params) {
-        ContentletVersionInfo cv;
+        Optional<ContentletVersionInfo> cv;
```

Comment:
Codacy found an issue: [Avoid declaring a variable if it is unreferenced before a possible exit point.](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=6214071)